docs/testing.md

```diff
@@ -78,37 +79,9 @@ When you are finished with the test, you can tear down the associated infrastruc
 #### Examples
 
     ansible-playbook pipelines/install_pipeline.yml -e forklift_state=up -e pipeline_os=debian10 -e pipeline_type=foreman -e pipeline_version=nightly
+    ansible-playbook pipelines/install_pipeline.yml -e forklift_state=up -e pipeline_os=centos7 -e pipeline_type=katello -e pipeline_version=nightly -e pipeline_proxy_delay=1
```

Comment:
What do you think about: 

 * pipeline_proxy_version_offset=1
 * pipeline_proxy_versions_back=1

Reply:
I like the version offset one, used it now.